pyproject.toml

```diff
@@ -9,12 +9,7 @@ authors = [
 [tool.poetry.dependencies]
 python = "^3.6.2"
 rpaframework = { path = "packages/main", extras = ["cv", "playwright", "aws"] }
-rpaframework-core = { path = "packages/core" }
-rpaframework-dialogs = { path = "packages/dialogs" }
 rpaframework-google = { path = "packages/google" }
-rpaframework-pdf = { path = "packages/pdf" }
-rpaframework-recognition = { path = "packages/recognition" }
-rpaframework-windows = { path = "packages/windows" }
 
```

Comment:
I removed these since they are already included by **main** and we would have package version resolution issues since the latest code in _master_ isn't necessarily compatible with all the pinned versions.

The docs should still compile with the latest **main** in _master_ (and its pins) + any other non-**main** referenced sub-package we have.